The required minimum Python version is too strict for unknown reason.

--- orca-47.3/meson.build.orig
+++ orca-47.3/meson.build
@@ -7,7 +7,7 @@
 gnome = import('gnome')
 i18n = import('i18n')
 
-python_minimum_version = '3.10'
+python_minimum_version = '3.9'
 python3 = python.find_installation('python3', required: true)
 if not python3.language_version().version_compare(f'>= @python_minimum_version@')
   error(f'Python @python_minimum_version@ or newer is required.')
